.github/workflows/build_with_conda.yml

```diff
@@ -8,8 +8,9 @@ jobs:
       matrix:
         include:
           - os: macos-latest
+            mongo_bin:  /Users/runner/micromamba-root/envs/arcticdb/bin
           - os: ubuntu-latest
-
+          mongo_bin:  /home/runner/micromamba-root/envs/arcticdb/bin
```

Comment:
This is an invalid syntax. What's the intention here?

Reply:
this outdated an not used anymore.